schemas/2016-03-01/Microsoft.Insights.json

```diff
@@ -0,0 +1,533 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2016-03-01/microsoft.insights.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "microsoft.insights",
+  "description": "microsoft insights Resource Types",
+  "resourceDefinitions": {
+    "alertrules": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "microsoft.insights/alertrules"
```

Comment:
We need to follow the casing convention. I suppose it should be "Microsoft.Insights/alertRules".